Formula/b/bornagain.rb

```diff
@@ -0,0 +1,263 @@
+class Bornagain < Formula
+  require "etc"
+
+  desc "Simulate and fit neutron and x-ray reflectometry and GISAS"
+  homepage "https://bornagainproject.org"
+
+  url "https://jugit.fz-juelich.de/mlz/bornagain.git",
+      tag: "v22.0", revision: "08e72eb7da6677fde2178a7b879686c469fb90b5", depth: 1
+
+  license "GPL-3.0-or-later"
+
+  depends_on "cmake" => [:build, "3.20"]
+  depends_on "fftw"
+  depends_on "gsl"
+  depends_on "libcerf"
+  depends_on "libformfactor" => "0.3.2"
+  depends_on "libheinz" => "2.0.1"
+  depends_on "libtiff"
+  depends_on "python@3"
+  depends_on "qt@6"
```

Comment:
It should use the latest version of both 
```suggestion
  depends_on "python"
  depends_on "qt"
```

Reply:
Currently, BornAgain can be built only with Qt6. So I hesitate to leave the Qt version unspecified.

Reply:
`qt@6` is an alias for `qt` so in a way you already are.

Reply:
Updated the formula.